src/V2/Actor.php

```diff
@@ -10,7 +10,10 @@ class Actor implements ActorInterface
     protected $NamespacePrefix;
     protected $NamespaceRelative;
     protected $RelativeClassPath;
-    protected $ParentRelativeClassPath;
+    protected $ParentActorRelativeClassPath;
```

Comment:
probably a not-for-now, but is v2 an opportunity to force typed properties?

Reply:
Agreed on both counts. There's a whole lot of internal cleanup that can/should happen, but this is currently more focused on the changes for the API / functionality improvements.